nnpdf_data/nnpdf_data/theory.py

```diff
@@ -0,0 +1,99 @@
+"""
+This module provides an unique source and definition for all the possible parameters
+that a theory card can contain.
+"""
+
+import dataclasses
+import logging
+
+DEPRECATED_KEYS = ["MaxNfAs", "SxRes", "SxOrd" "EScaleVar", "Qedref", "global_nx"]
+
+log = logging.getLogger(__name__)
+
+
+class TheoryCardError(Exception):
+    pass
+
+
+@dataclasses.dataclass(frozen=True)
+class TheoryCard:
+    ID: int  # ID number of the theory
+    PTO: int  # Perturbative order (0 = LO, 1 = NLO, 2 = NNLO ...)
+    FNS: str  # Flavor number scheme (e.g. FONLL-C)
+    DAMP: int  # Whether a damping function is applied or not for FONLL
+    IC: int  # 0 = perturbative charm only , 1 = intrinsic charm allowed
+    ModEv: str  # DGLAP evolution solution method (e.g. EXA or TRN)
+    XIR: float  # Renormalization scale over the hard scattering scale ratio
+    XIF: float  # Factorization scale over the hard scattering scale ratio
+    NfFF: int  # Number of active flavors, only for FFNS or FFN0 schemes
+    QED: int  # Max order of alpha_qed in the splitting functions
```

Comment:
```suggestion
    QED: int  # Max order of alpha_qed in the evolution
```
this will also impact the couplings